topology/probes/k8s/pod.go

```diff
@@ -63,6 +63,24 @@ func (h *podHandler) Map(obj interface{}) (graph.Identifier, graph.Metadata) {
 	return graph.Identifier(pod.GetUID()), m
 }
 
+func GetAppByPod(pod *v1.Pod) string {
+	return pod.Labels["app"]
+}
+
+func GetPodsByHosts(podCache *ResourceCache, hosts []string) []*v1.Pod {
+	pods := []*v1.Pod{}
+	for _, pod := range podCache.List() {
+		if pod, ok := pod.(*v1.Pod); ok {
+			for _, host := range hosts {
+				if host == pod.Labels["app"] {
```

Comment:
is "app" the default label used to designate host ?

Reply:
Yes